docs/antora.yml

```diff
@@ -0,0 +1,32 @@
+name: migrations 
+version: ~
+title: Neo4j Migrations Docs
+start_page: ROOT:index.adoc
+nav:
+  - modules/ROOT/nav.adoc
```

Comment:
Configure the navigation within your documentation component. Here's an example of the Kafka Integration Documentation (left menu is the navigation):

![image](https://user-images.githubusercontent.com/333276/198578794-e7ceda1e-55dc-42aa-9b7d-907f0df6206e.png)